### ISSUE TITLE
There's a bug when using a "Umfrage" 

### ISSUE THREAD

**Issue by Uatschitchun** [studio-arrenberg/quartiersplattform#134]
When participating in a survey on arrenberg.app, I get this:
![Screenshot_20220613_040852_com opera browser_1](https://user-images.githubusercontent.com/25088926/173268423-9d47d9fe-36c7-4b01-9f0a-8cc0e0491c8b.jpg)

**Comment by j0hannr** [comment]
@Uatschitchun um das zu Reproduzieren benötige ich einmal Angaben zum Gerät, Browser und wie dieser erzeugt wurde.

**Comment by Uatschitchun** [comment]
@j0hannr ähm, ja... Browser war garantiert Opera, ob die Fehlermeldung allerdings auf dem Handy oder Laptop kam!? Wenn Laptop, dann Linux, obgleich das irrelevant sein sollte. So ich mich recht erinnere, habe ich mich nur an einer Umfrage beteiligt...

**Comment by Uatschitchun** [comment]
@j0hannr
Gerade nochmal probiert. Wenn ich hier: https://arrenberg.app/projekte/quartiersplattform/ abstimmen will, wie mir die Quartiersplattform gefällt, erhalte ich den o.g. Fehler.
Browser ist Opera auf nem Android Handy.

**Comment by j0hannr** [comment]
@Uatschitchun Wird die stimme mit gezählt? und was passiert wenn die Seite neu geladen wird, bleibt die stimme bestehen?

**Comment by Uatschitchun** [comment]
@j0hannr nein, die Stimme wird nicht gezählt und auch neu laden ändert weder etwas am "count", noch an der Fehlermeldung.
Ich habe jetzt keine anderen Umfragen ausprobiert, um nicht Ergebnisse zu verfälschen. Haben Umfragen ein Ablaufdatum oder können sie eins haben?
